library/src/main/java/com/schibsted/spain/barista/assertion/BaristaVisibilityAssertions.kt

```diff
@@ -84,21 +85,45 @@ object BaristaVisibilityAssertions {
     withText(containsString(text)).assertAny(isDisplayed())
   }
 
+  @JvmStatic
+  fun assertContains(@StringRes stringId: Int) {
+    val resourceText = InstrumentationRegistry.getInstrumentation().targetContext.getString(stringId)
+    withText(containsString(resourceText)).assertAny(isDisplayed())
```

Comment:
To avoid duplicated code, you can use
`assertContains(resourceText)`

The same applies to the other new functions

Reply:
yes, I'll make these changes